content/blog/2021-08-24-pre-defined-models-exp.md

```diff
@@ -0,0 +1,148 @@
+---
+title: Using Experiments to Improve Pre-trained Models
+date: 2021-08-24
+description: |
+There are times it will be easier to take a pretrained model and fine-tune it to work with your data. You can do that with DVC experiments.
+descriptionLong: |
+When you're working with a pretrained model and you want to test different fine-tuning values, using DVC experiments can help you do that faster.
+picture: 2021-08-17/pretrained-models.png
+pictureComment: Using Experiments to Improve Pre-trained Models
+author: milecia_mcgregor
+commentsUrl: https://discuss.dvc.org/t/pretrained-model-experiments/727
+tags:
+  - MLOps
+  - Experiments
+  - Reproducibility
+  - DVC
+---
+
+## Intro
+
+There are plenty of machine learning models available that have been trained to
+solve one problem and the knowledge gained from that can be applied to a new,
+yet related problem. For example, a model like AlexNet has been trained on
+millions of images so you could potentially use this to classify cars, animals,
+or even people. This is called transfer learning and it can save a lot of time
+on developing a model from scratch.
+
+For us to take advantage of transfer learning, we can use fine-tuning to adopt
+the model to our new problem. In many cases, we start by replacing the last
+layer of the model. With the AlexNet example, this might mean the last layer was
+previously used to classify cars but our new problem is classifying animals.
+
+Even though we already have the bulk of the model defined, we'll still have to
+do some experimentation to determine whether we need to replace more layers in
+the model or if any other changes need to be made.
+
+In this post, we'll go through an example of fine-tuning
+[AlexNet](https://towardsdatascience.com/alexnet-the-architecture-that-challenged-cnns-e406d5297951)
+and
+[SqueezeNet](https://towardsdatascience.com/review-squeezenet-image-classification-e7414825581a)
+to classify bees and ants. We'll use DVC to handle experiments for us and we'll
+compare the results of both models at the end.
+
+## Initialize the pre-trained model
+
+We'll be fine-tuning the AlexNet model and the SqueezeNet model to classify
+images of bees and ants. You can find the project we're working with in
+[this repo](https://github.com/iterative/pretrained-model-demo).
+
+In the `pretrained-model-tuner.py` file, you'll find the code that defines both
+the AlexNet and SqueezeNet models. We start by initializing these models so we
+can get the number of model features and the input size we need for fine-tuning.
+
+Since the project has everything we need to initialize the models, we can start
+training and comparing the differences between them with the ants/bees dataset.
+Running experiments to get the best tuning for each model can make it difficult
+to see which changes led to a better result. That's why will be using DVC to
```

Comment:
insert "we"

Reply:
Last two sentences with minor modifications could make it to the `descpritionLong`, which will benefit from it a lot I think.